### ISSUE TITLE
Android apps never sleep

### ISSUE THREAD

**Issue by ddunsmuir** [part-cw/lambdanative#409]
Currently even the most basic Lambdanative Android app disables the screen timeout (is always considered active regardless of user action). Despite your Android OS screen timeout settings, if you run a Lambdanative app in the foreground the screen will stay on indefinitely even without any user action or plugin running. This appears to be the case on older Android and the newest.

**Comment by ddunsmuir** [comment]
@karliwalti and @0-8-15 you worked on making the applications not go to sleep to always stay active, right? Could that be related to this opposite issue? Were any of these efforts for Android or just other platforms?

**Comment by 0-8-15** [comment]
I don't think so.  I believe this behavior has always been like that, at least as long as I began to use it.
